### ISSUE TITLE
Code Questions

### ISSUE THREAD

**Issue by Twil-7** [Luke-Luo1/POPDG#2]
Thanks for your excellent work, but I have some small questions when learning your code.

In the original EDGE paper, when calculating the loss function, the weights of reconstruction loss, velocity loss, fk loss, and foot loss are: 0.636, 2.964, 0.646, 10.942. But in your work, the weights of reconstruction loss, velocity loss, fk loss, and body loss are: 0.636, 0.2964, 0.646, 1000.942.

Why does the weight of  velocity loss and body loss suddenly change so large? Is it a typo?

![image](https://github.com/user-attachments/assets/2f9a32df-1610-4576-b2c6-66633e5526fd)

**Comment by Luke-Luo1** [comment]
The adjustments to the loss function weights in our implementation were made to maintain the values of the different loss functions at comparable magnitudes during training. Since the PopDanceSet dataset differs from the AIST++ dataset, we found it necessary to adjust the coefficients accordingly.

**Comment by Luke-Luo1** [comment]
Thank you for your interest in our work and for your insightful question.

The adjustments to the loss function weights in our implementation were made to maintain the values of the different loss functions at comparable magnitudes during training. Since the PopDanceSet dataset differs from the AIST++ dataset, we found it necessary to adjust the coefficients accordingly.
